### ISSUE TITLE
[BUG] <title>Adding multiple entries of html causes entries to duplicate

### ISSUE THREAD

**Issue by chilkoot** [notepad-plus-plus/notepad-plus-plus#15178]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

Ok, I'm not a code guy...  I take a piece of plain text and make it html.  e.g. so I copy a <p> and paste it to the beginning of all the paragraphs.   I go back and look and maybe the first one I pasted now has 5 <p>, the next one 4, the next 3 ---   you get what I mean.  The <p> just happened to be the last one to do it on me.  it has happened to me with other html, I believe they were all html that duplicated but that is the only thing I would be copying like that.   I am using win 10 on a desktop computer.  No fancy apps  A spell corrector now that I think of it. It doesn't automatically change anything, it just highlights something.

### Steps To Reproduce

1.  
2.  
3.  
Haven't tried, to be honest, it just happens, I'm thinking not consistently

### Current Behavior

As I said, I start with plain text file, if I copy a piece of html into a file, say I do a <p> to 5 paragraphs in a row, I go back to the first one and there are 5 <p>.   I don't do it a lot but it happens & not necessarily that specific html, not necessarily the first of a paragraph.

### Expected Behavior

If I put one <p> in and go to the next paragraph and paste a second <p> in, the first one should still be by itself.

### Debug Information

```shell
Notepad++ v8.6.5   (64-bit)
Build time : Mar 29 2024 - 17:04:43
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 10 Home (64-bit
OS Version : 22H2
OS Build : 19045.4412
Current ANSI codepage : 1252
Plugins : 
    DSpellCheck (1.4.24)
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
```

### Anything else?

Sorry, I'm usually on a roll & just fix it and go on.   It might look like this @ the start of the first paragraph  <p><p><p><p><p>   It is not a big critical thing, it just caught me by surprise.  It has been happening (off and on, but I don't do this much) for maybe a couple of months, not sure about that.   just trying to help.  I don't do anything fancy, very basic html.   Here is the last file it did it on, but I don't think that has anything to do with a specific file.     www.dumbellnebula.com/RFK.htm

**Comment by alankilborn** [comment]
It sounds like maybe you are maybe ctrl+clicking when you place a caret and thus obtaining multiple simultaneous carets.  But your report is poor and hard to follow, so no one can be sure.  Bug reports are supposed to elicit exacting actions from you such that one can easily attempt to reproduce.
